spark/dl/src/main/scala/com/intel/analytics/bigdl/example/languagemodel/PTBModel.scala

```diff
@@ -21,6 +21,23 @@ import com.intel.analytics.bigdl.nn.Graph._
 import com.intel.analytics.bigdl.nn.{TimeDistributed, _}
 
 object PTBModel {
+  def transformer(
+     inputSize: Int = 10000,
+     hiddenSize: Int = 256,
+     outputSize: Int = 10000,
+     numLayers: Int = 2,
+     keepProb: Float = 2.0f)
+  : Module[Float] = {
+    val input = Input[Float]()
+    val transformer = Transformer[Float](vocabSize = inputSize,
+      hiddenSize = hiddenSize, numHeads = 4, filterSize = hiddenSize*4,
+      numHiddenlayers = numLayers, embeddingDropout = 1- keepProb,
+      attentionDropout = 0.1f, ffnDropout = 0.1f).inputs(input)
+    val linear = Linear[Float](hiddenSize, outputSize)
+    val output = TimeDistributed[Float](linear).inputs(transformer)
+    Graph(input, output)
+  }
+
   def apply(
```

Comment:
the above one is based on transformer, do we need to change this to lstm ?

Reply:
ok, change to lstm